fr/index.html

```diff
@@ -0,0 +1,159 @@
+  <style>
+    html {
+      height: 100%;
+    }
+    
+    body {
+      min-height: 100%;
+      display: grid;
+      grid-template-rows: 1fr auto;
+
+      margin: 0 auto;
+      max-width: 672px;
+      line-height: 1.6;
+      font-size: 18px;
+      color: #222;
+      background-color: #FAFAFA;
+      padding: 0 10px;
+      font-family: "Open Sans";
+    }
+
+    @media screen and (min-width: 680px) {
+      main {
+        margin: 40px 0;
+      }
+    }
+
+    h1, h2 {
+      line-height: 1.2;
+    }
+
+    footer {
+      grid-row-start: 2;
+      grid-row-end: 3;
+      color: #888;
+      font-size: 9px;
+    }
+
+    a {
+      color: #07A;
+    }
+    a:visited {
+      color: #941352;
+    }
+    footer > a {
+      color: #888 !important;
+    }
+
+    blockquote {
+      background: #F9F9F9;
+      border-left: 10px solid #CCC;
+      margin: 1.5em 10px;
+      padding: 0.5em 10px;
+
+      font-family: monospace;
+    }
+    blockquote p {
+      display: inline;
+    }
+
+    @media screen and (min-width: 400px) {
+      blockquote {
+        display: table;
+      }
+      blockquote > * {
+        display: table-cell;
+      }
+      blockquote > .name {
+        padding-right: 8px;
+      }
+    }
+  </style>
+</head>
+<body>
+  <main>
+    <h1>Ne demande pas la permission de demander, demande juste.</h1>
+    <p>
+      De temps en temps, dans des channels que je fréquente,
+      je vois quelqu'un apparaît et demander,
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Aucun expert Java dans le coin?
+      </p>
+    </blockquote>
+    <p>
+      C'est une mauvaise formulation pour plusieurs raisons. Ce que la personne est 
+      <em>vraiment</em> en train de demander,
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Aucun expert Java qui serait à prêt à s'engager à examiner mon problème,
+        quel qu'il soit, même s'il n'est pas relié à Java ou même si une personne
+        ne connaissant rien à Java pourrait répondre à ma question?
+      </p>
+    </blockquote>
+    <p>
+      Il a une multitude de raisons pour lesquelles les personnes AYANT la 
+      réponse ne répondraient pas. En demandant ça, vous demandez plus que ce que
+      vous pensez demander.
+    </p>
+    <p>
+      Vous demandez aux gens de prendre des responsabilités. Vous questionnez 
+      les gens sur la confiance qu'ils portent sur leurs compétences.
+      Vous écartez aussi inutilement les autres. Je réponds souvent aux questions
+      reliées à des langages ou des librairies que je n'ai jamais utilisées,
+      car la réponse relève (pour le développeur) du bon sens.
+    </p>
+    <p>
+      Vous pouvez aussi être vu comme quelqu'un...
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        J'ai une question à propos de Java mais je suis trop fatigué pour formuler
```

Comment:
is `fatigué` the correct word here ?. The user isn't tired he is just lazy

Reply:
hey, thanks for the review. I think both work. "Fatigué" means tired but can also be used to say "lazy" sometimes.